## Задание 1

Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде.
Результаты:
1)Число юзеров в группах, которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 2)Число юзеров в группах, которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 
3)Конверсии : conv1 = 1.05% conv2 = 1.47% .

Для оценки конверсии используется метод Хи-квадрат на однородность распределения в двух ген совокупностях или Z критерий долей.
В данной задаче будем использовать Z критерий долей. Необходимо посчитать доверительный интервал при alpha = 5%. Т.е. в 95% случаев данный интервал захватит истинное математическое ожидание.

Нулевая гипотеза – Между конверсией в двух группах нет статистически значимых различий.
Альтернативная гипотеза – Между конверсией в двух группах есть статистически значимые различия
alpha = 0.05
z(alpha/2) = z(0.025) = 1.96

In [3]:
import math

conv_1_size, conv_2_size = 164, 228
s_1_size, s_2_size = 15_550, 15_550
conv_1, conv_2 = conv_1_size/s_1_size, conv_2_size/s_2_size

In [4]:
var_1 = conv_1 * (1 - conv_1) / s_1_size
var_2 = conv_2 * (1 - conv_2) / s_2_size

x1 = (conv_2 - conv_1 - 1.96 * math.sqrt(var_1 + var_2)) * 100
x2 = (conv_2 - conv_1 + 1.96 * math.sqrt(var_1 + var_2)) * 100
print([x1, x2])

[0.16363911081042679, 0.6595120145915024]


Т.к. 0 не входит в доверительный интервал [0.16, 0.66], то на основе собранных данных существует статистически значимая разница между группами при alpha = 5%. Отвергается нулевая гипотеза. Изменение из второго теста необходимо применить.

## Задание 2

Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?


Для оценки конверсии используется метод Хи-квадрат на однородность распределения в двух ген совокупностях или Z критерий долей. Т.к. размер выборки большой (10000 элементов в каждой группе) предпочтительнее применить Z критерий долей